tests/quantization/test_compressed_tensors.py

```diff
@@ -161,10 +161,3 @@ def test_compressed_tensors_kv_cache(vllm_runner):
     with vllm_runner(model_path, kv_cache_dtype="fp8") as llm:
         output = llm.generate_greedy("Hello world!", max_tokens=20)
         assert output
-
-
-def test_compressed_tensors_fused_moe(vllm_runner):
```

Comment:
@ElizaWszola is there a reason we're disabling this test?

Reply:
I'll bring it back - accidentally deleted when cleaning up after merge